### DISCUSSION TITLE
BeReal API

### DISCUSSION THREAD

**Discussion by chemokita13** [rvaidun/berealviewer#109] [Show and tell]
Do you know were to learn the BeReal API? There is some docs or any repo? Thanks

**Comment by ronaldrupp** [comment]
There is currently no official BeReal API, but there is this [repo](https://github.com/notmarek/BeFake). There is also a good article about ["Hacking BeReal" ](https://blog.oscars.dev/posts/hacking_bereal_with_man_in_the_middle/). And of course you can reverse engineer BeReal and look into the network activity.

**Reply by chemokita13** [reply]
Thank you so much !!
